flow/designs/asap7/mock-array/config.py

```diff
@@ -7,16 +7,17 @@
 placement_grid_y     = 2.16
 
 # number of Elements in row and column, can be control by user via environment variable 
-# MOCK_ARRAY_TABLE (rows, cols, width, height, pitch_x, pitch_y)
+# MOCK_ARRAY_TABLE (rows, cols, width, height, space_x, space_y)
 #  rows, cols       - number of Element in rows, cols
 #  width, height    - width and height of each Element
+#  space_x, space_y - spacing between Element
 #
-# When the pitch is equal to the width/height, we have routing by abutment
+# When the space_? is equal to the 0, we have routing by abutment
 # https://en.wikipedia.org/wiki/Pitch#Linear_measurement
-#
-#  pitch_x, pitch_y - placement pitch for each Element, in x and y direction
+
+
 # specification are in unit of placement grid
-rows, cols, ce_x, ce_y, pitch_x, pitch_y = map(int, os.environ.get("MOCK_ARRAY_TABLE").split())
+rows, cols, ce_x, ce_y, space_x, space_y = map(int, os.environ.get("MOCK_ARRAY_TABLE").split())
```

Comment:
i prefer pitch to space

Reply:
Ok .. will change that back

Reply:
After a few try, space specification option seem a better choice

- pitch needs to know the size of Element
- pitch number must be precise to achieve 0 gap

Please let me know if you still prefer pitch.

Reply:
I think it is 6 of one or half a dozen of the other situation. *Something* has to be very precisely and accurately specified for elements to have it's pins lined up. I like pitch. Width and gap can then be imprecise...

Reply:
Ok ... will change the variable and the code ...